cores/esp8266/core_esp8266_phy.cpp

```diff
@@ -314,19 +305,43 @@ extern int IRAM_ATTR __wrap_spi_flash_read(uint32_t addr, uint32_t* dst, size_t
     return 0;
 }
 
-extern int __get_rf_mode(void)  __attribute__((weak));
+extern int __get_rf_disable_mode(void) __attribute__((noinline, weak));
+extern int __get_rf_disable_mode(void)
+{
+    // Starting from arduino core v3: wifi is disabled at boot time
+    //  mode == 4: Disable RF after deep-sleep wake up, just like modem sleep; this has the least current
+    //  consumption; the device is not able to transmit or receive data after wake up.
+    return 4;
+}
+
+extern int __get_rf_powerup_disable_mode(void) __attribute__((noinline, weak));
+extern int __get_rf_powerup_disable_mode(void)
+{
+    // Starting from arduino core v3: wifi is disabled at boot time
+    // mode == 2: RF initialization only calibrates VDD33 which will take about 2 ms;
+    // this has the least current consumption.
+    return 2;
+}
+
+extern int __get_rf_mode(void)  __attribute__((noinline, weak));
```

Comment:
Are these numbers mentioned somewhere in the docs, or is the rtos sdk the source of these?
(...citation needed?)

Reply:
That's 3.3.12. system_phy_set_powerup_option from the API Reference.

Reply:
The latter. Please propose how to do this.

Reply:
I can't really find a pre-existing and successful pattern for this in the sources' comments - sometimes SDK references end up pointing at documents that are no longer there - or the one's in the right place (Espressif server?) make no mention of what previously people were referring to. So, "the numbers" are explained in this code section and that's it :-)

Reply:
Direct quote of the 3.3.12?
Window to fetch 2.2.2 docs from espressif is already lost (?), but I wonder why that document was not saved as a reference directly in the repo